JustSaying/JustSayingFluently.cs

```diff
@@ -385,4 +404,10 @@ public IMayWantOptionalSettings WithAwsClientFactory(Func<IAwsClientFactory> aws
             return this;
         }
     }
+
+    public enum TopicQueryBehaviour
+    {
+        DisableExistenceCheck,
+        NoPreload

```

Comment:
it's not clear to me that "DisableExistenceCheck" and "NoPreload" form an enumeration of possible behaviours. 
Is "Preload" the same as "EnableExistenceCheck"

Both are framed as negatives "don't do x", which is generally not what you want from var names.